plugins/kotlin-as-java/src/main/kotlin/jvmOverloads.kt

```diff
@@ -0,0 +1,12 @@
+package org.jetbrains.dokka.kotlinAsJava
+
+import org.jetbrains.dokka.model.Annotations
+import org.jetbrains.dokka.model.Documentable
+import org.jetbrains.dokka.model.properties.WithExtraProperties
+import org.jetbrains.kotlin.utils.addToStdlib.firstNotNullResult
+
+internal fun WithExtraProperties<out Documentable>.jvmOverloads(): Annotations.Annotation? =
```

Comment:
I'd personally call it something like `hasJvmOverloads` just to be more specific